src/grammar.js

```diff
@@ -53,9 +55,8 @@ const grammar = ohm.grammar(`
     string
       = "\\"" (~"\\"" ~"\\n" any)* "\\""
 
-    special
-      = id | value | variable | wildcard | hole | space
-
+    word_char = ~("#" | "\\"" | "$" | "_" | space | digit | ".") any
```

Comment:
By getting rid of special, we need to keep two lists of definitions of characters in sync. Not a huge deal, but is there some ohm magic we could use to keep that connection? something closer to
```
word
  = ~special any
  | (~special any)+ "."+
```
This is unfamiliar territory for me so I'm not sure if my question makes sense

Reply:
maybe `symbol` or `mark`